### DISCUSSION TITLE
Select of multiple text in Winedt

### DISCUSSION THREAD

**Discussion by athmarg** [WinEdt-Team/WinEdt-Support#60] [1.1 General Help and Discussion]
I want to select a large number of words in my text in order to apply to each one of them the \emph{...} style .. is it possible to select more than words in one step instead of selecting each one of them, one word at a time? this is very time consuming and ... painful .. thanks a lot !! Athanasios

**Comment by urmel1960** [comment]
If I have understood you correctly - I would use search and replace. Replace the corresponding word with the LaTeX-formatted word. There is an option to select "all".
